docs/_legacy/app-checks-reference.md

```diff
@@ -397,7 +397,7 @@ See [ZDE004](https://zapier.com/developer/documentation/v2/app-checks-reference/
 
 <a name="ZDE019"></a>
 
-## ZDE019 - Important Polling Triggers Should Always Have Sample Data
+## ZDE019 - Visible Polling Triggers Should Always Have Sample Data
```

Comment:
This check is legacy, so likely is not actually applying anymore. However, looking at the actual code, it seems to only affect Visible triggers, not just Important triggers, so I updated the text anyway.